stumpy/core.py

```diff
@@ -679,15 +706,30 @@ def rolling_nanstd(a, w):
     w : numpy.ndarray
         The rolling window size
 
+    welford : bool, default False
+        When False (default), the computation is parallelized and the std of
```

Comment:
replace `std` with `stddev`

Reply:
Thank you for the suggestion. I will fix this.